src/navigator-card-body.jsx

```diff
@@ -292,7 +292,45 @@ export const NavigatorCardBody = ({
         }
     };
 
-    const handleClick = (ev, file) => {
+    const getFilenameForEvent = event => {
+        let data_item = null;
+        let elem = event.target;
+        // Limit iterating over all parents
+        for (let i = 0; i < 10; i++) {
+            data_item = elem.getAttribute("data-item");
+            if (data_item)
+                break;
+
+            if (elem.parentElement)
+                elem = elem.parentElement;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-249-20240305-140311-42a1f07c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)